app/Http/Controllers/EmployeesController.php

```diff
@@ -33,20 +33,24 @@ public function displayImage($fileName)
 
     public function list()
     {
-        $employees = Employee::all()->sortBy(function($employee) {
-            // Teilen Sie den Namen durch Leerzeichen in Teile auf und geben Sie den letzten Teil (Nachname) zurück
-            $lastNames = explode(' ', $employee->name);
-            return Str::lower(end($lastNames)); // Stellen Sie sicher, dass Sie Zeichenfolgen in Kleinbuchstaben vergleichen
-        });
-    
+        // $employees = Employee::all()->sortBy(function($employee) {
+        //     // Teilen Sie den Namen durch Leerzeichen in Teile auf und geben Sie den letzten Teil (Nachname) zurück
+        //     $lastNames = explode(' ', $employee->name);
+        //     return Str::lower(end($lastNames)); // Stellen Sie sicher, dass Sie Zeichenfolgen in Kleinbuchstaben vergleichen
+        // })->simplePaginate(10);
+
+        // Order by the name of Employee and adding a simplePaginate (max 10 employees for page)
+        $employees = Employee::orderBy('name')->simplePaginate(10);
```

Comment:
Was this a requirement? now the names are not sorted by Last name any more

Reply:
If it is a requirement to order them by the last name, I will try to maintain the previous logic. The reason I commented on it and changed it was because the pagination did not work with sortBy.

Reply:
yes, please try to maintain the last name sorting if possible